common/lib/xmodule/xmodule/templates/html/zooming_image.yaml

```diff
@@ -2,28 +2,26 @@
 metadata:
     display_name: Zooming Image Tool
 data: |
-      <h3 class="hd hd-2">Zooming Image Tool</h3>
+      <h3 class="hd hd-3">Zooming Image Tool</h3>
       <p>Use the Zooming Image Tool to enable learners to see details of large, complex images.</p>
       <p>With the Zooming Image Tool, the learner can move the mouse pointer over a part of the image to enlarge it and see more detail.</p>
-      <p>To use the Zooming Image Tool, you must first add the <a href="http://files.edx.org/jquery.loupeAndLightbox.js" target="_blank">jquery.loupeAndLightbox.js JavaScript file</a> to your course.</p>
+      <p>To use the Zooming Image Tool, you must first add the <a href="http://files.edx.org/image-zoom-tool.js" target="_blank">image-zoom-tool.js JavaScript file</a> to your course.</p>
       <p>You must also add both the regular and magnified image files to your course.</p>
       <p>The following HTML code shows the format required to use the Zooming Image tool. For the example in this template, you must replace the values in <i>italics</i>.</p>
+      <p>Your small and large image should have the same aspect ratio.</p>
```

Comment:
I think it would make sense to include this warning in the text on line 9--

> You must also add both the regular and magnified image files to your course, and they must have the same aspect ratio.